packages/daemon/src/formula-identifier.js

```diff
@@ -1,24 +1,68 @@
+/// <ref types="ses">
+
 const { quote: q } = assert;
 
+const idPattern =
+  /^(?:(?<type>endo|readable-blob|worker|pet-store|pet-inspector|eval|lookup|make-unconfined|make-bundle|host|guest|handle|peer|directory|least-authority|loopback-network):)?(?<number>[0-9a-f]{128}):(?<node>[0-9a-f]{128})|(?<type2>web-bundle):(?<number2>[0-9a-f]{32})(?<node2>:[0-9a-f]{128})$/;
```

Comment:
The last `web-bundle` capture group includes the final `:`.
```suggestion
const idPattern =
  /^(?:(?<type>endo|readable-blob|worker|pet-store|pet-inspector|eval|lookup|make-unconfined|make-bundle|host|guest|handle|peer|directory|least-authority|loopback-network):)?(?<number>[0-9a-f]{128}):(?<node>[0-9a-f]{128})|(?<type2>web-bundle):(?<number2>[0-9a-f]{32}):(?<node2>[0-9a-f]{128})$/;
```